### DISCUSSION TITLE
heron main branch won't compile with latest bevy main branch

### DISCUSSION THREAD

**Discussion by Niedzwiedzw** [jcornaz/heron#157] [Q&A] [answered]
error: failed to select a version for `syn`.
    ... required by package `heron_macros v0.12.1 (https://github.com/jcornaz/heron?rev=3636e07#3636e075)`
    ... which satisfies git dependency `heron_macros` of package `heron v0.12.1 (https://github.com/jcornaz/heron?rev=3636e07#3636e075)`
    ... which satisfies git dependency `heron` of package `guild-mayhem v0.1.0 (/home/niedzwiedz/Programming/guild-mayhem)`
versions that meet the requirements `^1.0` are: 1.0.81

all possible versions conflict with previously selected packages.

  previously selected package `syn v1.0.65`
    ... which satisfies dependency `syn = "=1.0.65"` of package `bevy v0.5.0`
    ... which satisfies dependency `bevy = "^0.5.0"` of package `heron v0.12.1 (https://github.com/jcornaz/heron?rev=3636e07#3636e075)`
    ... which satisfies git dependency `heron` of package `guild-mayhem v0.1.0 (/home/niedzwiedz/Programming/guild-mayhem)`

**Comment by jcornaz** [comment, marked as answer]
Hi,

Heron's tracks released versions of bevy only. I do not provide support for its main branch. For that you have to fork heron, fix the problems yourself and use your fork.

We'll update `heron` when bevy `0.6` will be published. (@zicklag  already started a PR in that direction: #134)
